etna/models/nn/chronos/base.py

```diff
@@ -0,0 +1,288 @@
+import os
+import warnings
+import zipfile
+from abc import abstractmethod
+from pathlib import Path
+from typing import Dict
+from typing import List
+from typing import Sequence
+from typing import Union
+from urllib import request
+
+import pandas as pd
+
+from etna import SETTINGS
+from etna.datasets import TSDataset
+from etna.distributions import BaseDistribution
+from etna.libs.chronos import BaseChronosPipeline
```

Comment:
Don't we need to wrap this into `if SETTINGS.chronos_required`? It seems like we done this for embeddings.